environments/staging.yaml

```diff
@@ -3,4 +3,12 @@ continuumLogin:
 
 continuumAdaptor:
   port: 3001
-  continuumApiUrl: "http://continuumtest--gateway.elife.internal"
\ No newline at end of file
+  continuumApiUrl: "http://continuumtest--gateway.elife.internal"
+
+submission:
+  scienceBeam:
+    apiUrl: "https://sciencebeam-texture.elifesciences.org/api/convert"
```

Comment:
intention is to replace this with a sidecar or another deployment later? This is a `demo` environment so it will work for `staging` in terms of monitoring, SLA, etc

Reply:
not sure

Reply:
it's in `staging` so re-evaluating when there is `prod`